packages/amplify_datastore/example/ios/unit_tests/DataStoreHubEventStreamHandlerTests.swift

```diff
@@ -63,7 +63,7 @@ class DataStoreHubEventStreamHandlerTests: XCTestCase {
         waitForExpectations(timeout: 1.0)
         // cancellation needed to make sure that Hub token is invalidated to
         // prevent collisions between tests
-        hubHandler.onCancel(withArguments: nil)
+        let _ = hubHandler.onCancel(withArguments: nil)
```

Comment:
Q: what is the purpose of this change? Is this the norm in swift or enforced via linting?

Reply:
Swift linter complains about the value was not bing used. This is a common pattern to satisfy the linter rule, while denoting the return value is not used. Relevant [SO](https://stackoverflow.com/a/32656172)